CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ## dev
 
 ### `Added`
-
+- [#117](https://github.com/nf-core/sarek/pull/117) - Add `Trim Galore` possibilities to Sarek
```

Comment:
[#117](https://github.com/nf-core/sarek/pull/117) modified CHANGELOG.md to fix double `Added` error